ui/package.json

```diff
@@ -0,0 +1,144 @@
+{
+  "name": "libretime-ui",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "serve": "vue-cli-service serve",
+    "build": "vue-cli-service build",
+    "test:unit": "vue-cli-service test:unit --testMatch '**/*.spec.js' --setupFiles ./src/test.setup.js",
+    "lint": "vue-cli-service lint",
+    "i18n:report": "vue-cli-service i18n:report --src './src/**/*.?(js|vue)' --locales './src/locales/**/*.json'",
+    "serve:storybook": "start-storybook -p 6006"
+  },
+  "dependencies": {
+    "axios": "^0.21.1",
+    "core-js": "^3.8.1",
+    "dayjs": "^1.10.7",
+    "howler": "^2.2.3",
+    "v-calendar": "^2.1.4",
+    "vue": "^2.6.11",
+    "vue-blob-json-csv": "^0.1.7",
+    "vue-i18n": "^8.17.3",
+    "vue-router": "^3.2.0",
+    "vuetify": "^2.4.0"
+  },
+  "devDependencies": {
+    "@babel/preset-react": "^7.12.10",
+    "@intlify/vue-i18n-loader": "^2.0.0-beta.2",
+    "@mdi/font": "^5.8.55",
+    "@storybook/addon-a11y": "^6.1.11",
+    "@storybook/addon-actions": "^6.1.11",
+    "@storybook/addon-docs": "^6.1.11",
+    "@storybook/addon-essentials": "^6.1.11",
+    "@storybook/addon-viewport": "^6.1.11",
+    "@storybook/addons": "^6.1.11",
+    "@storybook/vue": "^6.1.11",
+    "@vue/cli-plugin-babel": "~4.5.0",
+    "@vue/cli-plugin-eslint": "~4.5.0",
+    "@vue/cli-plugin-router": "~4.5.0",
+    "@vue/cli-plugin-unit-jest": "~4.5.0",
+    "@vue/cli-service": "^4.5.0",
+    "@vue/test-utils": "^1.1.2",
```

Comment:
I think it would be a good idea. The scaffolding was done by @hairmare; I guess he was more comfortable with Vue CLI (which ships Webpack) so that's what we're currently running with. If we are to start again with Vue3 it might be easier to scaffold with Vite.

Reply:
I've been trying to integrate everything with Vite and it just isn't working. Even using templates from Github haven't gotten me anywhere, especially since Vuetify 3 is still in alpha and relatively unstable. Webpack isn't a deal-breaker but it's crazy slow, especially as we develop.

Reply:
And what with this ? https://github.com/vitejs/vite/issues/305

Reply:
Honestly, all of this stuff is over my head. Again, why I'm trying to open it up for more people to work on.